commander/src/generators/base_generator.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import Generator from 'yeoman-generator';
+import Storage from 'yeoman-generator/lib/util/storage';
+import { join, dirname } from 'path';
+import { BootstrapGeneratorOptions } from '../types';
+
+// const DEFAULT_TEMPLATE_NAME = 'lisk-ts';
+
```

Comment:
if not used, then we can delete?

Reply:
Its uncommented and used in the second PR.